configs/default.cfg

```diff
@@ -275,6 +275,9 @@ cooler_0_target_temp = 60
 # different temperature:
 # therm-e-fan-0-target_temp = 70
 
+# Sets pwm frequency used for controlling fans
```

Comment:
Could you add a note that this only applies to the B* revision boards?

Reply:
did some mention on g-code "help" section